docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst

```diff
@@ -83,7 +83,140 @@ are:
++-----------------+---------------+------------+-----------------------------------+
+| Addition of     | Choice        | Yes, but   |                                   |
+| Choice          |               | they must  |                                   |
+| Parameters      |               | be         |                                   |
+|                 |               | optional   |                                   |
+|                 |               |            |                                   |
++-----------------+---------------+------------+-----------------------------------+
+| Deletion of     | Choice        | No         |                                   |
+| Choice          |               |            |                                   |
+| Parameters      |               |            |                                   |
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Choice        | No         |                                   |
+| Consuming       |               |            |                                   |
+| choice to       |               |            |                                   |
+| Non-Consuming   |               |            |                                   |
+| Choice, or      |               |            |                                   |
+| vice-versa      |               |            |                                   |
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Choice        | Yes        |                                   |
+| Choice Body     |               |            |                                   |
++-----------------+---------------+------------+-----------------------------------+
+| Change of       | Choice        | Yes,       | You can change a user data type   |
+| Choice return   |               | except for | (record, variant, enum) contained |
+| type            |               | tuple or   | in the return type. In this case  |
+|                 |               | scalar     | you can add a field to the record |
+|                 |               | types      | or a new case to the              |
+|                 |               |            | variant/enum.                     |
+|                 |               |            |                                   |
+|                 |               |            | In particular: you cannot change  |
+|                 |               |            | a collection, Optional, List, or  |
+|                 |               |            | Map; you can change Tuple, (to a  |
+|                 |               |            | triple) if you define your own    |
+|                 |               |            | type.                             |
+|                 |               |            |                                   |
```

Comment:
Make capitalization consistent in this list

Reply:
Changed